be/engine.c

```diff
@@ -814,6 +814,10 @@ M0_INTERNAL int m0_be_engine_start(struct m0_be_engine *en)
 	if (rc != 0)
 		return M0_ERR(rc);
 
+	rc = be_log_header_generation_inc(&(en->eng_log), &(en->eng_log.lg_header));
```

Comment:
parens are redundant:
```c
rc = be_log_header_generation_inc(&en->eng_log, &en->eng_log.lg_header);
```

Reply:
When uncertain, I always look at `man 7 operator` for C operators precedence reference.

Reply:
Can this function return failure?
If yes, do we need to cleanup before return err?